architecture-building-systems/CityEnergyAnalyst
# Automated generation of streets and district file from Open Street Maps for CEA

This solves issue #1849 and #1851  which introduces three new tools for CEA

To test:

Run 
datamanagement/district-helper.py
datamanagemnet/streets-helper.py
datamanagemnet/terrain-helper.py

It should create respectively the district.shp, the streets.shp and the terrain.tiff for your scenario>

Then run radiation, so to check that the geometry works.
